src/kv/LevelDBStore.cc

```diff
@@ -270,11 +270,13 @@ int LevelDBStore::get(const string &prefix,
   assert(value && (value->length() == 0));
   utime_t start = ceph_clock_now(g_ceph_context);
   int r = 0;
-  KeyValueDB::Iterator it = get_iterator(prefix);
-  it->lower_bound(key);
-  if (it->valid() && it->key() == key) {
-    value->append(it->value_as_ptr());
-  } else {
+  leveldb::ReadOptions options;
+  std::string value;
```

Comment:
Great. I don't know how to grab this commit? could you please send me a link? thanks!